angular/projects/spark-angular/src/lib/components/inputs/sprk-radio-group/sprk-radio-group.component.ts

```diff
@@ -0,0 +1,127 @@
+import {
+  Component,
+  AfterContentInit,
+  Input,
+  ContentChild,
+  ContentChildren,
+  QueryList,
+} from '@angular/core';
+import { uniqueId } from 'lodash';
+import { SprkRadioItemComponent } from '../sprk-radio-item/sprk-radio-item.component';
+import { SprkFieldErrorDirective } from '../../../directives/inputs/sprk-field-error/sprk-field-error.directive';
+import { SprkLabelDirective } from '../../../directives/inputs/sprk-label/sprk-label.directive';
+
+@Component({
+  selector: 'sprk-radio-group',
+  template: `
+    <div [ngClass]="getClasses()" [attr.data-id]="idString">
+      <ng-content select="[beforeSprkRadios]"></ng-content>
+      <fieldset *ngIf="label; else sprkRadios" class="sprk-b-Fieldset">
```

Comment:
I thought we were going to not have radioGroup handle fieldsets or legends

Reply:
haha good catch, knew I was forgetting something. This is updated now. I made two new directives for sprkFieldset and sprkLegend.